js/sites.js

```diff
@@ -72,11 +72,12 @@
         aIndex = this.cell.index;
       }
 
-      gNewTab.sendToBrowser("NewTab:PinLink", {
+      gNewTab.sendToBrowser("NewTab:GetUpdatePages", {
```

Comment:
actually I'm wrong. the newtab-dev code seems to do that already with `let outerWindowID = tabbrowser ? tabbrowser.selectedBrowser.outerWindowID : null;`